### ISSUE TITLE
Build of Perl-5.28.0-GCCcore-7.3.0.eb fails in singularityhub/centos:latest

### ISSUE THREAD

**Issue by jose-d** [easybuilders/easybuild-easyconfigs#7007]
It seems there is some dependency missing in 

[Perl-5.28.0-GCCcore-7.3.0.eb](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/Perl/Perl-5.28.0-GCCcore-7.3.0.eb)

Build in container with singularityhub/centos:latest with additional packages 

`yum-utils epel-release ohpc-base-compute lmod-ohpc EasyBuild-ohpc GitPython pysvn graphviz vim zlib-devel libyaml-devel openssl-devel gdbm-devel readline-devel ncurses-devel libffi-devel curl git redis libxml2-devel libxslt-devel libcurl-devel libicu-devel python libuuid-devel
`
and

```
yum -y groupinstall "Development tools"
yum-builddep -y perl
```

is failing in build of **perl-Net-SSLeay** installation step:

```
== 2018-10-12 13:03:28,777 run.py:192 INFO running cmd: tar xzf /sw/local/el7/x86_64/sources/p/Perl/extensions/Net-SSLeay-1.85.tar.gz 
== 2018-10-12 13:03:28,808 run.py:192 INFO running cmd:  perl Makefile.PL PREFIX=/sw/local/el7/x86_64/software/lang/Perl/5.28.0-GCCcore-7.3.0  
== 2018-10-12 13:03:28,999 run.py:192 INFO running cmd:  make -j 64  
== 2018-10-12 13:03:29,048 build_log.py:158 ERROR EasyBuild crashed with an error (at ?:124 in __init__): cmd " make -j 64 " exited with exit code 2 and output:
make: *** No targets specified and no makefile found.  Stop.

```
It is enough to install build-deps of SSLeay to have it working..

`yum-builddep -y perl-Net-SSLeay
`
I work on problem isolation, however I already suspect that we should include [ExtUtils::MakeMaker](https://metacpan.org/pod/ExtUtils::MakeMaker) somewhere into [Perl-5.28.0-GCCcore-7.3.0.eb](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/Perl/Perl-5.28.0-GCCcore-7.3.0.eb).

**Comment by jose-d** [comment]
> `yum-builddep -y perl-Net-SSLeay`
..nope. https://www.fzu.cz/~jose/temp/easybuild-Perl-5.28.0-20181012.151005.RieWg.log

**Comment by jose-d** [comment]
```
Singularity container.img:~/projects/container_for_eb_testing> rpm -qa | grep openssl
openssl-devel-1.0.2k-12.el7.x86_64
openssl-libs-1.0.2k-12.el7.x86_64
Singularity container.img:~/projects/container_for_eb_testing>
```
fails and..
```
Singularity container.img:~/projects/container_for_eb_testing> rpm -qa | grep openssl
openssl-libs-1.0.2k-12.el7.x86_64
openssl-1.0.2k-12.el7.x86_64
openssl-devel-1.0.2k-12.el7.x86_64
Singularity container.img:~/projects/container_for_eb_testing>
```
is OK. so dependency to openssl and libs should be added. I'll take look.

**Comment by boegel** [comment]
@jose-d Are you recipes for the container(s) you're using for testing available somewhere?

I've been wanted to set up something like this for a while now, but haven't found time yet to flesh it out fully...

**Comment by boegel** [comment]
Closing this, fixed in #7008, thanks @jose-d!
